pkgs/by-name/ja/jack-passthrough/package.nix

```diff
@@ -26,22 +27,28 @@ stdenv.mkDerivation {
     meson
     ninja
   ];
+
   buildInputs = [
     fmt_9
     libjack2
   ];
 
-  meta = with lib; {
+  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };
+
+  meta = {
     description = "Simple app to help with JACK apps that behave strangely";
     longDescription = ''
       Creates a JACK passthrough client with an arbitrary name and number of
       ports. Common uses include tricking stubborn applications into creating
       more ports than they normally would or to prevent them from
       auto-connecting to certain things.
     '';
-    # license unknown: https://github.com/guysherman/jack-passthrough/issues/2
-    license = licenses.unfree;
-    maintainers = [ maintainers.PowerUser64 ];
+    #
```

Comment:
Could probably remove this whole line. (pointing to line 46 with the empty comment)

Reply:
Must've forgot :p